From 6e8048311f854184ae5f16c822e6f6f0fd122e54 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Fri, 11 Dec 2015 14:59:10 +0800
Subject: [PATCH] MLK-11996 ARM: imx: fix ddr_type config error

The config option for getting DDR type should
be 'HAVE_IMX_MMDC' and 'HAVE_IMX_DDRC'. Otherwise,
get_ddr type will always return 0(DDR3).

Signed-off-by: Bai Ping <b51503@freescale.com>
---
 arch/arm/mach-imx/common.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-imx/common.h b/arch/arm/mach-imx/common.h
index cb3873f..71e4d53 100644
--- a/arch/arm/mach-imx/common.h
+++ b/arch/arm/mach-imx/common.h
@@ -181,12 +181,12 @@ int imx6q_set_lpm(enum mxc_cpu_pwr_mode mode);
 void imx6q_set_int_mem_clk_lpm(bool enable);
 void imx6sl_set_wait_clk(bool enter);
 void imx6_enet_mac_init(const char *compatible);
-#ifdef CONFIG_HAVE_MMDC
+#ifdef CONFIG_HAVE_IMX_MMDC
 int imx_mmdc_get_ddr_type(void);
 #else
 static inline int imx_mmdc_get_ddr_type(void) { return 0; }
 #endif
-#ifdef CONFIG_HAVE_DDRC
+#ifdef CONFIG_HAVE_IMX_DDRC
 int imx_ddrc_get_ddr_type(void);
 #else
 static inline int imx_ddrc_get_ddr_type(void) { return 0; }
-- 
1.8.0

